### PR TITLE
Documented Quay Bridge Operator per PROJQUAY-335

### PR DESCRIPTION
**Pull request by chrisnegus**
Documented the Quay Bridge Operator.

### CODE REVIEW

File: modules/proc_quay-bridge-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: QuayIntegration
+metadata:
+  name: example-quayintegration
+spec:
+  clusterID: openshift  <1>
+  credentialsSecretName: openshift-operators/quay-integration <2>
+  quayHostname: https://<QUAY_URL>   <3>
+  whitelistNamespaces: <4>
+    - default
```
Comment (dongboyan77) [inline modules/proc_quay-bridge-operator.adoc:201]: The `whitelistNamespaces` is optional, bridge operator will sync all namespaces to Quay except openshift prefix project if do not set it

File: modules/proc_quay-bridge-operator.adoc [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  insecureRegistry: false <5>
+----
+<1> The clusterID value should be unique across the entire ecosystem.
+This value is optional and defaults to openshift.
+<2> For credentialsSecretName, replace `openshift-operators/quay-integration`
+with the name of the namespace and the secret containing the token you created earlier.
+<3> Replace QUAY_URL with the hostname of your {productname} instance.
+<4> The white listed namespace (default, in this example) will now have an
+associated {productname} organization. Use any namespace you like here
```
Comment (dongboyan77) [inline modules/proc_quay-bridge-operator.adoc:210]: Should add above comment here

File: modules/proc_quay-bridge-operator.adoc (hunk 1 above)
Comment (chrisnegus) [reply modules/proc_quay-bridge-operator.adoc:201]: Okay. I wrote this: "The whitelistNamespaces is optional. If not used, the Bridge Operator will
sync all namespaces to {productname} except the openshift prefixed project.
In this example, the white listed namespace (default) will now have an
associated {productname} organization. Use any namespace you like here."

File: modules/proc_quay-bridge-operator.adoc (hunk 2 above)
Comment (chrisnegus) [reply modules/proc_quay-bridge-operator.adoc:210]: Done.

File: modules/proc_quay-bridge-operator.adoc [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+spec:
+  clusterID: openshift  <1>
+  credentialsSecretName: openshift-operators/quay-integration <2>
+  quayHostname: https://<QUAY_URL>   <3>
+  whitelistNamespaces: <4>
+    - default
+  insecureRegistry: false <5>
+----
+<1> The clusterID value should be unique across the entire ecosystem.
+This value is optional and defaults to openshift.
```
Comment (dongboyan77) [inline modules/proc_quay-bridge-operator.adoc:205]: The clusterID should not collision if multiple OCP clusters use the same Quay as registry

**Comment by dongboyan77** [comment]
If Quay configured with self-signed certificate, user need add the certificate into OCP, refer to https://docs.openshift.com/container-platform/4.2/builds/setting-up-trusted-ca.html

**Comment by dongboyan77** [comment]
@chrisnegus I left comments after you merging

**Pull request merged**